pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/buffer/BytesTest.java

```diff
@@ -56,6 +56,19 @@ void toStringWorks1() {
         assertEquals("54686973206973206d7920537472696e6721", Bytes.wrap(s).toString());
     }
 
+    @Test
+    void testReplicate() {
+        byte[] arr = new byte[] { 0, 1, 2 };
+
+        // Only wrap the last two elements:
+        Bytes bytes = Bytes.wrap(arr, 1, 2);
+        Bytes replicatedBytes = bytes.replicate();
+
+        assertEquals(2, replicatedBytes.length());
+        assertEquals(arr[1], replicatedBytes.getByte(0));
+        assertEquals(arr[2], replicatedBytes.getByte(1));
+    }
```

Comment:
Should it also test that changing bytes in the original array doesn't affect the replicated object?

Reply:
Good idea. Just pushed a commit with this improvement.